flake.nix

```diff
@@ -312,6 +362,8 @@
               pkgs.shellcheck
               pkgs.taplo
               pkgs.yamlfmt
+              pkgs.gomplate
+              pkgs.html-tidy
```

Comment:
Doesn't look like the check_all.sh script was changed, so these shouldn't be needed.